image/docker/schema2/test/test_manifest.py

```diff
@@ -293,8 +287,8 @@ def test_get_schema1_manifest():
                 {"created": "2018-04-12T18:37:09.284840891Z", "created_by": "bar"},
             ],
         },
-        "sha256:b5b2b2c507a0944348e0303114d8d93aaaa081732b86451d9bce1f432a537bc7",
-        1885,
+        CONFIG_DIGEST,
+        CONFIG_SIZE,
```

Comment:
Were these changes inferred by Flake8? Neat if so.

Reply:
Almost, flake8 complained about `\/` in the context where it should be just `/` (as the original json has `\/` that is interpreted as `/`). I could incorrectly replace it with `\\/`, but I decided to fix it, so it has changed its size and digest.